test/e2e/gardenadm/hightouch/gardenadm.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("gardenadm high-touch scenario tests", Label("gardenadm", "high
 		})
 
 		It("should initialize as control plane node", func(ctx SpecContext) {
-			stdOut, _, err := execute(ctx, 0, "gardenadm", "init", "-d", "/gardenadm/resources")
+			stdOut, _, err := execute(ctx, 0, "gardenadm", "--log-level=debug", "init", "-d", "/gardenadm/resources")
```

Comment:
Why? :)

Reply:
I found it useful to see the debug logs of the `flow` package, including task start times and run durations when investigating test failures. I typically use this during local development, too, and debug logs shouldn't hurt in CI.
Do you object?

Reply:
No, I was just wondering about the motivation/whether there was a special reason. lgtm